### PR TITLE
update capabilities

### PR DESCRIPTION
**Pull request by underhood**

### CODE REVIEW

File: proto/nodeinstance/info/v1/info.proto [hunk 1]
```diff
@@ -45,12 +45,9 @@ message UpdateNodeInfo {
...
 
-  NodeInstanceInfo node_instance_info = 10;
-  // info specific to the node_instance for this node available trough agent
-  // who sends this message.
-  // e.g. machine learning is enabled for this node and processing is done
-  // by the actual agent (parent). (child itself might or might not be
-  // ml ml_capable by itself (see node_info))
+  //NodeInstanceInfo node_instance_info = 10;
+  // moved to UpdateNodeInstanceConnection
+  reserved 10;
```
Comment (papazach) [inline proto/nodeinstance/info/v1/info.proto:50]: Why remove it from here?

Comment (underhood) [reply proto/nodeinstance/info/v1/info.proto:50]: Should it be in both places? Seems like duplicate to me... I can put it back

Comment (papazach) [reply proto/nodeinstance/info/v1/info.proto:50]: Yeap I suggest to put it back for now, should assess if other stuff would misbehave if this is absent which is something we cannot deal with currently.

Comment (underhood) [reply proto/nodeinstance/info/v1/info.proto:50]: ok, will do it
my general feeling is however we are adding workaround on top of workaround

**Comment by underhood** [comment]
removed the removal of the now duplicate nodeinstance capabilities in NodeInfo message

**Pull request merged**